### PR TITLE
Add zap as a payment option

### PR DESCRIPTION
**Pull request by daallen303**

### CODE REVIEW

**Review by Neeraj-Ghodla — APPROVED** [review]

File: src/transacting.md [hunk 1]
```diff
@@ -124,6 +124,8 @@ is returned with the payment request info as `payments[].ledger`.
 | epay.nzd.main      | epay.nzd.main            | *Centrapay asset id*           |
 | pocketvouchers     | g.pocketvouchers.pv      | *Pocket Vouchers voucher code* |
 | bitcoin.main       | g.crypto.bitcoin.mainnet | *Bitcoin transaction id*       |
+| cennznet.main      | cennznet.main            | *Cennznet transaction id*      |
+| zap.main           | zap.main                 | *Zap transaction id*           |
```
Comment (ncjones) [inline src/transacting.md:128]: I think this should read "Waves transaction id"

File: src/merchants.md [hunk 2]
```diff
@@ -292,6 +292,7 @@ See [Merchant configuration model](#merchant-configuration-model) for expected r
 |:--------------------------------------------|:-------|:-------------------------------------------------------------------------------------|
 | {% break . paymentOptions[\*].walletId %}   | String | Merchant's Centrapay wallet to receive payments. Required for `centrapay.nzd` types. |
 | {% break . paymentOptions[\*].terminalId %} | String | Merchant's Epay terminal id. Required for `epay.nzd` types.                          |
+| {% break . paymentOptions[\*].zapAddress %} | String | Merchant's public zap address. Required for `zap.main` types.                        |
```
Comment (ncjones) [inline src/merchants.md:295]: I think this should be "wavesAddress" and "Merchant's public Waves address".

File: src/merchants.md [hunk 3]
```diff
@@ -308,4 +309,5 @@ The following table describes the supported payment option types.
 | epay.nzd.test      | EPay NZ giftcards (test ledger)      |
 | cennznet.main      | Cennznet                             |
 | pocketvouchers     | Pocket Vouchers                      |
+| zap.main           | Zap                                  |
```
Comment (ncjones) [inline src/merchants.md:312]: "Zap tokens"

File: src/merchants.md (hunk 2 above)
Comment (daallen303) [reply src/merchants.md]: what about `wavesPublicAddress` just to be more precise?

**Review by maticko — APPROVED** [review]

**Pull request merged**